### PR TITLE
Smooth turns documentation

### PR DESCRIPTION
**Pull request by bresch**
replaces #811

### CODE REVIEW

**Comment by bkueng** [comment]
For reference: this is also exposed in qgc: https://github.com/mavlink/qgroundcontrol/pull/9533

File: en/flight_modes/mission.md [hunk 1]
```diff
@@ -124,33 +125,38 @@ Unless otherwise noted, the implementation is as defined in the MAVLink specific
 * [MAV_CMD_DO_VTOL_TRANSITION](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_VTOL_TRANSITION)
 * [MAV_CMD_NAV_DELAY](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_DELAY)
 * [MAV_CMD_NAV_RETURN_TO_LAUNCH](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_RETURN_TO_LAUNCH)
-* [MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET)
-* [MAV_CMD_DO_SET_ROI_NONE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI_NONE)
-
+* [MAV_CMD_DO_CONTROL_VIDEO](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_CONTROL_VIDEO)
```
Comment (hamishwillee) [inline en/flight_modes/mission.md:128]: @julianoes Below is a set of new mission items I saw added while doing this PR. I think I was thorough  but do you happen to know of any more?

File: en/flight_modes/mission.md [hunk 2]
```diff
@@ -124,33 +125,38 @@ Unless otherwise noted, the implementation is as defined in the MAVLink specific
...
   
 ## Inter-Waypoint Trajectory
 
 PX4 expects to follow a straight line from the previous waypoint to the current target (it does not plan any other kind of path between waypoints - if you need one you can simulate this by adding additional waypoints).
 
 MC vehicles will change the *speed* when approaching or leaving a waypoint based on the [jerk-limited](../config_mc/mc_jerk_limited_type_trajectory.md#auto-mode) tuning.
+The vehicle will follow a smooth rounded curve towards the next waypoint (if one is defined) defined by the acceptance radius ([NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD)).
+The diagram below shows the sorts of paths that you might expect.
 
+![acc-rad](../../assets/flying/acceptance_radius_mission.png)
```
Comment (hamishwillee) [inline en/flight_modes/mission.md:155]: @bresch Just FYI, I moved the image up to hear (out of bullet point). This is where we are talking about the effect of the radius.

File: en/flying/missions.md [hunk 3]
```diff
@@ -27,6 +27,23 @@ By default this is the next waypoint.
...
+For a multi-rotor drones, the acceptance radius is tuned using the parameter [NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD).
+By default, the radius is small to ensure that multirotors pass above the waypoints, but it can be increased to create a smoother path such that the drone starts to turn before reaching the waypoint.
+
+The image below shows the same mission flown with different acceptance radius parameters:
+
+![acceptance-radius-comparison](../../assets/flying/acceptance_radius_comparison.png)
+
+The speed in the turn is automatically computed based on the acceptance radius (= turning radius) and the maximum allowed acceleration and jerk (see [Jerk-limited Type Trajectory for Multicopters](../config_mc/mc_jerk_limited_type_trajectory.md#auto-mode)).
+
+:::tip
```
Comment (hamishwillee) [inline en/flying/missions.md:43]: FYI, @bresch This is all good. All I have really done here is push add the note below that cross links to your other docs.

**Comment by hamishwillee** [comment]
Merging. Thanks @bresch . I did a subedit, and there are some notes - but they are just "FYI"

File: en/flying/missions.md (hunk 3 above)
Comment (bresch) [reply en/flying/missions.md:43]: Cool, thanks!

**Pull request merged**